src/NServiceBus.Autofac/AutofacBuilder.cs

```diff
@@ -8,6 +8,10 @@ namespace NServiceBus
     /// <summary>
     /// Autofac Container
     /// </summary>
+    [ObsoleteEx(
+          Message = "Support for Autofac is provided via the NServiceBus.Extensions.DependencyInjection package.",
```

Comment:
I think it's a bit confusing that the message is stating it like it's always been this way, without any regard for what has changed. Something like "Support for external dependency injection containers is no longer provided by NServiceBus adapters for each container library. Instead, the NServiceBus.Extensions.DependencyInjection library provides the ability to use any container that conforms to the Microsoft.Extensions.DependencyInjection container abstraction.

Sorry, I always think longer deprecation messages are better. I would also hope the exact text of the message would be highly googleable somewhere in our documentation.

Reply:
I like it, adjusted